wire-library/wire-grpc-java-server/src/test/java/com/squareup/wire/kotlin/grpcserver/AddImplBaseKtTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.squareup.wire.kotlin.grpcserver
```

Comment:
can we do golden tests instead of just asserting that a few lines match? Is there precedence in this repo?

Reply:
The precedence in Kotlin and Java generators is to check that a few lines match. I agree though, it would be much better to do golden files. I'll go add them.